content/docs/help/faq.md

```diff
@@ -48,6 +48,26 @@ Of course biscuit is a recent piece of tech, which makes it harder to justify th
 
 Consider unionizing to protect yourself from being unduly fired.
 
+## How can biscuit be integrated with OAuth and OIDC?
```

Comment:
I would rephrase this making it clear that there are 3 types of tokens and biscuit fits only one. I remember learning oauth and mixing all the types of token together, specially the identity and access tokens. Something like:

You can find 3 different types of tokens on Oauth and oidc, biscuit only fit in one:
- access token: Biscuit is focused on the authorization and delegation side of API access, so it can be used as an OAuth access token. OAuth does not mandate a specific format for the access token.
- refresh token: ...
- identity token: oidc mandates the identity token should be a jwt...

Reply:
there's an open issue for a more in-depth discussion of oauth / oidc #14